src/main/java/page_objects/ChangePasswordPage.java

```diff
@@ -0,0 +1,37 @@
+package page_objects;
+
+import common.element.Button;
+import common.element.Label;
+import common.element.TextBox;
+import model.User;
+import org.openqa.selenium.By;
+
+public class ChangePasswordPage extends GeneralPage{
+  private final TextBox txtCurrentPassword = new TextBox(By.cssSelector("#currentPassword"));
+  private final TextBox txtNewPassword = new TextBox(By.cssSelector("#newPassword"));
+  private final TextBox txtConfirmPassword = new TextBox(By.cssSelector("#confirmPassword"));
+  private final Button btnChangePassword = new Button(By.cssSelector("[title='Change password']"));
+  private final Label lblChangePassword = new Label(By.cssSelector(".message"));
+  private final Label lblConfirmPasswordErrorMessage = new Label(By.cssSelector(".confirm-password label:last-child"));
```

Comment:
this xpath is not good, don't use position to capture xpath if there's a better way

Reply:
I fixed it. Thank you.